src/services/pdf.js

```diff
@@ -11,15 +11,15 @@ export const createPDF = async lessonPlan => {
   var html = ['<body style="font-family:Arial">'];
 
   // add title
-  html.push(header(`Lesson Plan: ${lessonPlan['lessonPlanName']}`, 1));
+  html.push(header(`Lesson Plan: ${lessonPlan.lessonPlanName}`, 1));
 
   html.push(moduleInformation(lessonPlan['Warm Up'], 'Warm Up'));
   html.push(moduleInformation(lessonPlan['Main Lesson'], 'Main Lesson'));
   html.push(moduleInformation(lessonPlan['Cool Down'], 'Cool Down'));
   // eslint-disable-next-line no-extra-boolean-cast
-  if (!!lessonPlan['Notes']) {
+  if (!!lessonPlan.Notes) {
```

Comment:
@ramyzhang Does this screw anything up? Linting automatically changed it to this format (from `!!lessonPlan['Notes']` to `!lessonPlan.Notes`) but I can change it back

Reply:
yap just tested it and it's fine still!! i was just copying baker's syntax tbh